csrc/mmdeploy/codebase/mmseg/segment.cpp

```diff
@@ -41,40 +45,60 @@ class ResizeMask : public MMSegmentation {
                      mask.shape());
       return Status(eNotSupported);
     }
-    if (!with_argmax_) {
-      MMDEPLOY_ERROR("TODO: SDK will support probability featmap soon.");
+    if ((mask.data_type() != DataType::kFLOAT) && !with_argmax_) {
+      MMDEPLOY_ERROR("probability feat map only support float32 output");
       return Status(eNotSupported);
     }
 
+    auto channel = (int)mask.shape(1);
     auto height = (int)mask.shape(2);
     auto width = (int)mask.shape(3);
     auto input_height = preprocess_result["img_metas"]["ori_shape"][1].get<int>();
     auto input_width = preprocess_result["img_metas"]["ori_shape"][2].get<int>();
+    if (!with_argmax_) {
+      // (C, H, W) -> (H, W, C)
+      ::mmdeploy::operation::Context ctx(device_, stream_);
+      std::vector<int> axes = {0, 2, 3, 1};
+      permute_.Apply(mask, mask, axes);
```

Comment:
IMO, CHW format score maps are more intuitive for the users.

Reply:
And this also contradicts with the fact that the model's result is in NCHW format.

Reply:
CHW format may be inconvenient to use opencv resize if the user want to resize the output feature map.